src/stores/connectors/keys/get.go

```diff
@@ -0,0 +1,30 @@
+package keys
+
+import (
+	"context"
+	"github.com/consensys/quorum-key-manager/src/stores/store/entities"
+)
+
+func (c Connector) Get(ctx context.Context, id string) (*entities.Key, error) {
+	logger := c.logger.With("id", id)
+
+	key, err := c.db.Keys().Get(ctx, id)
```

Comment:
is this query excluding soft_deleted items ?

Reply:
adding tests

Reply:
tests added